app/views/search/_variable_search_results.html.erb

```diff
@@ -0,0 +1,54 @@
+<div class="pagination-summary">
+  <%= page_entries_info variables, entry_name: 'record' %>
+</div>
+
+<table id="search-results">
+  <thead>
+    <tr>
+      <th class="w20">Name</th>
+      <th class="w40">Long Name</th>
+      <th class="w15">Provider</th>
+      <th class="w15">Last Modified</th>
+    </tr>
+  </thead>
+  <tbody>
+
+    <% errors.each do |error| %>
+      <tr>
+        <td colspan="4">
+          <strong>Error:</strong> <%= error %>
+        </td>
+      </tr>
+    <% end %>
+
+    <% if variables.blank? %>
+      <tr>
+        <td colspan="4">
+          No variables found.
+        </td>
+      </tr>
+    <% else %>
+
+      <% variables.each do |variable| %>
+        <tr class='<%= cycle("alt", "") %>'>
+          <td>
+            <%= link_to variable['umm']['Name'], variable_path(variable['meta']['concept-id']) %>
+          </td>
+          <td>
+            <%= variable['umm']['LongName'] %>
+          </td>
+          <td>
+            <%= variable['meta']['provider-id'] %>
+          </td>
+          <td>
+            <% fragments = variable['meta']['revision-date'].split('T')
+               date = fragments[0].nil? ? 'UNKNOWN' : fragments[0] %>
```

Comment:
I kept this as it had been implemented for collections, but I will implement a view helper for these.